daprdocs/content/en/java-sdk-docs/java-client/_index.md

```diff
@@ -17,6 +17,31 @@ If you haven't already, [try out one of the quickstarts]({{< ref quickstarts >}}
 
 [Complete initial setup and import the Java SDK into your project]({{< ref java >}})
 
+## Initializing the client
+You can initialize a Dapr client as so:
+
+```java
+DaprClient client = new DaprClientBuilder().build()
+```
+
+This will connect to the default Dapr gRPC endpoint `localhost:50001`.
+
+
+#### Environment variables:  
+
+##### Dapr Sidecar Endpoints
+You can use the standardized `DAPR_GRPC_ENDPOINT` and `DAPR_HTTP_ENDPOINT` environment variables to
+specify a different gRPC endpoint. When these variables are set, the client will automatically use them to connect to the Dapr sidecar.
```

Comment:
Think we want both in this sentence since we have both endpoint env vars
```suggestion
specify a different gRPC or HTTP endpoint. When these variables are set, the client will automatically use them to connect to the Dapr sidecar.
```